app/MMB/src/scenes/tripServices/hotelMenuItem/HotelMenuItem.js

```diff
@@ -43,21 +55,46 @@ class HotelMenuItem extends React.Component<PropsWithContext, State> {
     this.props.navigation.navigate('MMBMainSwitchStack');
   };
 
-  openHotelsModal = (cityId: string, cityName: string) => {
+  sanitizeDates = (dates: {|
+    +checkin: string | null,
+    +checkout: string | null,
+  |}) => {
+    const checkin =
+      dates.checkin === null
+        ? DateFormatter(DateUtils.getUTCToday()).formatForMachine()
+        : dates.checkin;
+    const checkout =
+      dates.checkout === null
+        ? DateFormatter(
+            DateUtils(new Date(checkin)).addDays(1),
+          ).formatForMachine()
+        : dates.checkout;
+    return {
+      checkin,
+      checkout,
+    };
+  };
+
+  openHotelsModal = ({ cityId, cityName, ...rest }: HotelData) => {
```

Comment:
Imho, it would be clearer that you de-structure checkin and checkout.

Reply:
you are probably right, to name more variables 😞 

I will fix it 😄